sbin/anti-evil-maid-dump-evt-log

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euo pipefail -o errtrace
+
+source anti-evil-maid-lib
+
+range=$(xl dmesg | grep "SLAUNCH: reserving event log" | sed "s#.*(\(.*\))#\1#")
+base=$(($(echo "$range" | cut -d " " -f 1)))
+size=$(($(echo "$range" | cut -d " " -f 3) - base))
+
+printf "Found event log at %#x, size %#x (%d) bytes\n" $base $size $size
+
+validatetpm || exit 1
+
+dd if=/dev/mem bs=1 skip=$base count=$size of=/tmp/log.bin
```

Comment:
Ouch... I presume there is no better way to access it?

Reply:
You mean use of `/dev/mem` or having to write to an intermediate file?

Unless we want to add a Linux driver for TXT event log (and wait who knows how long for it to propagate), I don't know of any easier way than reading from memory directly.

As for intermediate file, it is described in commit message. Seems that `awk` requires `fseek()`-able input when reading whole file at once.

Reply:
`/dev/mem` of course.